corehq/apps/domain/management/commands/make_superuser.py

```diff
@@ -39,7 +39,7 @@ def handle(self, username, **options):
             logger.info("✓ User {} exists".format(couch_user.username))
         else:
             password = self.get_password_from_user()
-            couch_user = WebUser.create(None, username, password)
+            couch_user = WebUser.create(None, username, password, created_by=None, created_via=__name__)
```

Comment:
What do you think about forcing the user to supply their username here (which could be validated by prompting for their password and checking with `user.check_password(password)`).

Reply:
will take it as a follow up if thats okay?